packages/components/slot/src/Slot.doc.mdx

```diff
@@ -74,3 +90,10 @@ The Slot-n-Child defined `style` prop merging object technique is the `join` pro
 In case of collision, it will preserve the most declarative (child value)
 
 <Canvas of={SlotStories.Ref} />
+
+### Append
+
+A `Slot` can have more than 1 descendant elements as a children but only 1 of them **must** be an `Slottable`.
```

Comment:
A Slot can have multiple descendant elements as children, but only one of them **can** be a `Slottable`?

Reply:
```suggestion
A `Slot` can have multiple descendant elements as children, but only one of them can be a `Slottable`.
```